src/internal/m365/collection/exchange/collection.go

```diff
@@ -459,10 +481,13 @@ func (col *lazyFetchCollection) streamItems(
 			&lazyItemGetter{
 				userID:       user,
 				itemID:       id,
+				category:     col.FullPath().Category(),
```

Comment:
nit: please use `col.Category()` since it handles the case where `FullPath() == nil`